.env.example

```diff
@@ -7,3 +7,7 @@ COGNITE_PROJECT=
 COGNITE_BASE_URL=
 COGNITE_STATE_DB=
 COGNITE_STATE_TABLE=
+COGNITE_EXTRACTION_PIPELINE=
+LAKEHOUSE_BASE_URL=
+LAKEHOUSE_TABLE_NAME=
+LAKEHOUSE_ABFSS_PREFIX=
```

Comment:
As discussed in Teams, should probably converge on using either BASE_URL or ABFSS_PREFIX, but having both is confusing.